model/eligibility/Eligibility.php

```diff
@@ -147,6 +157,7 @@ public function jsonSerialize()
             'delivery' => $this->getDelivery()->getUri(),
             'testCenter' => $this->getTestCenter()->getUri(),
             'testTakers' => $this->getTestTakers(),
+            'proctored'=> $this->getProctored(),
```

Comment:
It should also reflect that the value is a boolean, so what about `isProctored`?